src/screens/home/components/benefits.js

```diff
@@ -75,6 +82,86 @@ class Benefits extends React.Component {
             </RadiumLink>
           </div>
         </div>
+
+        <div className="Benefits-demos">
+          <div className="Grid Grid--gutters Grid--full medium-Grid--fit">
+            <div className="Grid-cell u-textCenter">
+              <div className="Benefits-demo">
+                <DemoCustomChart />
+                <p className="SubHeading">
+                  <a
+                    href="./guides/custom-charts"
```

Comment:
All relative/internal links should use react-router’s `<Link>` so that in production, the basename `/open-source/victory/` gets applied!